.github/workflows/test-action.yml

```diff
@@ -0,0 +1,53 @@
+name: Test Pipelines Execute
+
+on:
+  workflow_dispatch:
```

Comment:
Should we do this on `pull_request` as well?

Reply:
Also, bugger, this org/repo can't use GHA (yet).  I'll look into our options for that short term tomorrow.

Reply:
> Should we do this on `pull_request` as well?

Absolutely! We might, however, end up with duplicate runs for the same commit if a pull request is open from that branch. This might work:
```
on:
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
    tags:
      - 'v*'
```

Reply:
We might also have tweak the `Extract branch name` step to handle PRs, tags and branches correctly.

Reply:
I changed the triggers to this:
```
on:
  workflow_dispatch:
  pull_request:
    branches:
      - main
  push:
    branches:
      - "**"
```
This way contributors get immediate feedback before creating a PR and PR's themselves get tested. Don't think tags necessarily need to be tested, if they're created from stable `main`. Also, running the tests with tags would most likely require a lot of work with the refs we use when invoking the action (e.g. `infra_live_repo_branch: ${{ steps.extract_branch.outputs.branch }}`)